src/gsp.py

```diff
@@ -0,0 +1,9 @@
+"""Get GSP boundary data from eso """
+import geopandas as gpd
+from nowcasting_dataset.data_sources.gsp.eso import get_gsp_shape_from_eso
+
+
+def get_gsp_boundaries_from_eso() -> gpd.GeoDataFrame:
```

Comment:
as per #50, I think I would also prefer this to be returned in lat/lon coordinates

Reply:
Thanks, ill force it to return lat lon in this api

Reply:
If `get_gsp_boundaries_from_eso` always returns WGS84 coordinates, then maybe consider making that really explicit in the function name by changing the function name to something like `get_lat_lon_gsp_boundaries_from_eso`?

Reply:
I would prefer 'get_latitude_longitude_gsp_boundaries_from_eso' to be fully verbose. 

Perhaps it would be enough to comment in that [function](https://github.com/openclimatefix/nowcasting_dataset/blob/main/nowcasting_dataset/data_sources/gsp/eso.py#L43)

Reply:
`get_latitude_longitude_gsp_boundaries_from_eso` im happy with that. Ill make an issue